src/server/utilities/mvtHeader/enabledExperimentsList.js

```diff
@@ -5,7 +5,7 @@ const enabledExperimentList = [
     pageTypes: ['STY', 'IDX'],
   },
   {
-    name: 'full_stack_test',
+    name: 'test_2',
     services: ['pidgin'],
     pageTypes: ['STY', 'IDX'],
```

Comment:
Similar here, may need changed to just `ARTICLE`

Reply:
Done, although page type casing is weird, we have it as lower case `article`